pkg/apis/controlplane/types.go

```diff
@@ -436,3 +436,64 @@ type EgressGroupList struct {
 	metav1.ListMeta
 	Items []EgressGroup
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type SupportBundleCollection struct {
+	metav1.TypeMeta
+	metav1.ObjectMeta
+	ExpiredAt      metav1.Time
+	SinceTime      string
+	FileServer     BundleFileServer
+	Authentication BundleServerAuthConfiguration
+}
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// SupportBundleCollectionList is a list of SupportBundleCollection objects.
+type SupportBundleCollectionList struct {
+	metav1.TypeMeta `json:",inline"`
+	metav1.ListMeta `json:"metadata,omitempty" protobuf:"bytes,1,opt,name=metadata"`
+	Items           []SupportBundleCollection `json:"items" protobuf:"bytes,2,rep,name=items"`
+}
+
+type BundleFileServer struct {
```

Comment:
Can we reuse the CRD definition?

Reply:
Updated to reuse CRD definition for `BundleFileServer`.